frontend/src/flashcards/NewCard.js

```diff
@@ -0,0 +1,26 @@
+import FlashcardFront from './FlashcardFront';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import newCardIcon from '../SVG/create-24px.svg';
+
+const cardStyle = {
+	width: '300px',
+	height: '350px',
+	margin: '10px',
+};
+
+/**
+ * A card that, when clicked on, would lead the user to the /CreateCard page
+ */
+class NewCard extends React.Component {
+	render() {
```

Comment:
Formatting — these are tabs, not spaces.

Reply:
fixed.